rule/rule.go

```diff
@@ -97,6 +102,16 @@ func LoadWorkspaceFile(path, pkg string) (*File, error) {
 	return LoadWorkspaceData(path, pkg, data)
 }
 
+// LoadMacroFile is similar to LoadFile but parses the file as a bzl
+// file.
+func LoadMacroFile(path, pkg, function string) (*File, error) {
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	return LoadMacroData(path, pkg, function, data)
+}
+
```

Comment:
We may also want an `EmptyMacroFile` to handle the case where the file doesn't already exist.

Reply:
Fixed and tested with TestImportReposFromDepToEmptyMacro

Reply:
It looks like `gofmt` might be needed here. GH is showing one space indent instead of tabs.